docs/sources/examples/mongodb/Dockerfile

```diff
@@ -7,8 +7,8 @@ MAINTAINER Docker
 
 # Installation:
 # Import MongoDB public GPG key AND create a MongoDB list file
-RUN apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv 7F0CEB10
-RUN echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | tee /etc/apt/sources.list.d/10gen.list
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+RUN echo "deb http://repo.mongodb.org/apt/ubuntu "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
```

Comment:
Couple of things
- you are correct, sudo is not required, have removed
- Dockerfile and documentation got out of sync, my mistake. Have corrected that
- The $(lsb_release...) is lifted directly from the Debian install instructions from mongodb.org, see http://docs.mongodb.org/manual/tutorial/install-mongodb-on-debian/
I think it makes sense for these instructions to line up with the documentation from MongoDB - if you feel strongly about this, happy to make the change.